js/src/personal/manny/components/ClickableImage.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from './ClickableImage.module.css'
+import PersonalImage from '../assets/PicOfManny.jpg'
+
+export function ClickableImage() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.imageContainer}>
+        <img
+          src={PersonalImage}
+          alt="Manny"
+          className={styles.personalImage}
+        />
+      </div>
+      <div className={styles.overlay}>
+        <div className={styles.menu}>
+          <div
+            className={styles.menuItem}
+            style={{ transform: 'rotate(-20deg) translateX(-50%)' }}
+          >
+            {'About Me'}
+          </div>
+          <div
+            className={styles.menuItem}
+            style={{ transform: 'rotate(-10deg) translateX(-30%)' }}
+          >
+            {'Resume'}
+          </div>
+          <div
+            className={styles.menuItem}
+            style={{ transform: 'rotate(0deg) translateX(-30%)' }}
+          >
+            {'Projects'}
+          </div>
+          <div
+            className={styles.menuItem}
+            style={{ transform: 'rotate(15deg) translateX(-73%)' }}
```

Comment:
just curious, why are some items spaced by different degrees (eg 5deg, 10deg, 15deg) ? Is it related to the text itself so if the letters like `g` and `l` happen to line up, we need more space between them?

Reply:
if possible could you offer more clarification on this